roles/libstatic/tasks/main.yml

```diff
@@ -25,6 +48,21 @@
   become: true
   notify: restart apache
   with_items: '{{ named_sites + sites }}'
+  changed_when: false
+  tags:
+    - install_sites
+
+- name: Grant permissions on deploy user
```

Comment:
This task looks really similar to the one that starts on line 16.  Does it really need to be run twice?

Reply:
It does, because when the files are cloned they are owned by the root user, and we have to re-set all the permissions in order to make the child files and folders have the correct permissions.

This PR follows the pattern in ezproxy and drupal - (`playbooks/roles/ezproxy/tasks/main.yml`, `playbooks/roles/drupal/tasks/main.yml`)